src/components/Judging/Admin/GradeTable.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
```

Comment:
is there any way we can just reuse the project table instead of creating an entire component here?

Reply:
made an abstract version lmk what u think